modules/installation-installing-bare-metal.adoc

```diff
@@ -67,6 +67,24 @@ INFO It is now safe to remove the bootstrap resources
 The command succeeds when the Kubernetes API server signals that it has been
 bootstrapped on the control plane machines.
 
+. Check that all the configured bare-metal hosts are available in the cluster:
++
+----
+oc get bmh -A
```

Comment:
Hi @rohennes, please note that apart of including worker nodes in the example here, which makes no sense for bootstrap process, this documentation is for UPI BM, AFAIK, oc get bmh is a command that will not return any resource in this case.

Reply:
Thanks Pedro. How about I move this `oc get bmh -A ` content to the following section: 

https://deploy-preview-45677--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#ipi-install-troubleshooting-following-the-installation_ipi-install-installation-workflow

Do you think that would be acceptable? This section is under **Deploying installer-provisioned clusters on bare metal** so it might be accurate here?

Reply:
Thanks Pedro. How about I move this `oc get bmh -A ` content to the following section: 

https://deploy-preview-45677--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#ipi-install-troubleshooting-following-the-installation_ipi-install-installation-workflow

Do you think that would be acceptable? This section is under **Deploying installer-provisioned clusters on bare metal** so it might be accurate here?

Reply:
Hi @rohennes, yes, I think that section would be more appropriate but please note that IPI installation has no separated user-based interaction between `wait-for bootstrap-complete` and `wait-for install-complete`, so not sure how to properly redact the content there to fit the `oc get bmh` example, maybe in parallel once the bootstrap completes?